prepare-testdata.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/sh
+set -e
+
+download() {
+    local from to
+    from="$1"
+    to="$2"
+
+    if [ ! -e "$to" ]; then
+        curl -fsSL -o "$to" "$from"
+    fi
+}
+
+error() {
```

Comment:
How this fn used?

Reply:
/dev/kvm check was there, and I forgot to remove the error function when I moved that from here to Makefile. Let me remove...

Reply:
https://github.com/firecracker-microvm/firecracker-go-sdk/pull/144/commits/3721c6919236d1c695c712e1bb082cf79bedb404 doesn't have the function.